projects/vdk-heartbeat/src/vdk/internal/heartbeat/vdk_heartbeat_data_job/trino/requirements.txt

```diff
@@ -1 +1,2 @@
+Faker
```

Comment:
Isn't vdk-trino considered "external" dependency, anyways?

Reply:
Yes. But I am adding Faker to explicitly check for that.